api-definitions/db_positions/api.yaml

```diff
@@ -0,0 +1,292 @@
+swagger: "2.0"
+info:
+  version: 1.0.1
+  title: Debtor Positions Interfaces API
```

Comment:
API = interface

```suggestion
  title: Debtor Positions API
```

Reply:
is `Debtor Positions` a term that makes sense for english speaking countries?

Reply:
I agree with you @balanza 👍 
I think that in some cases perhaps it's better to leave the original italian names,  if there isn't such thing in english

Reply:
I'm a big fan of using the very same terms you would use in informal discussions with stakeholders, even if they are in italian. 

Anyway, in our context it may be valuable to think of our projects as if they could be bootstrapped in countries other than Italy. In this sense, I think we should take an extra time to figure out a proper term.